src/main/java/com/dynatrace/oneagent/sdk/impl/OneAgentSDKFactoryImpl.java

```diff
@@ -30,13 +30,10 @@
 	 * OneAgent), increase oneSdkFix only.
 	 */
 	static final int oneSdkMajor = 1;
-	static final int oneSdkMinor = 5;
+	static final int oneSdkMinor = 7;
 	static final int oneSdkFix = 0;
 
-	/** the only SDK instance (one for every classLoader) */
-	private static OneAgentSDK oneSDK = null;
-
-	public static boolean debugOneAgentSdkStub = false;
+	public static final boolean debugOneAgentSdkStub = Boolean.parseBoolean(System.getProperty("com.dynatrace.oneagent.sdk.debug", "false"));
```

Comment:
What if someone sets this system property to an invalid boolean value, e.g. "on"? Won't this result in an user facing exception?

Reply:
default is false :)